### ISSUE TITLE
[bug] Paraglide init fails to initialise in new Project.

### ISSUE THREAD

**Issue by LorisSigrist** [inlang/monorepo#1617]
### Problem

Running `npx @inlang/paraglide-js@latest init` in a new project fails because `.inlang/plugin.inlang.messageformat/messages.json` does not exist. 

Adding an empty json file at that path solves the issue.

### Expected behavior

One of two options:
- a) The messageformat plugin should be able to cope with a missing file and not blow up
- b) The init command should automatically add the file at the expected path.

### Reproduction

1. Create a new SvelteKit project (or whatever) & install dependencies
2. git init
3. run `npx @inlang/paraglide-js@latest init` -> it blows up

### Other information

Computer: M1 Macbook Air.

Error message:
```
ℹ Creating a new inlang project in the current working directory.                                                                                        

 WARN  Failed to create a new inlang project.                                                                                                             

This is likely an internal bug. Please file an issue at https://github.com/inlang/monorepo.


 ERROR  An error occured in loadMessages() caused by Error: ENOENT: no such file or directory, open '/Users/---/dev/paraglide-init-error-reproduction/.inlang/plugin.inlang.messageFormat/messages.json'.

  at /Users/---/.npm/_npx/b3fd06ce0fca9d0c/node_modules/@inlang/sdk/dist/loadProject.js:105:50
```

Package-Manager: PNPM

Exact version:
```
dependencies:
  '@inlang/paraglide-js':
    specifier: 1.0.0-prerelease.7
    version: 1.0.0-prerelease.7
```

**Comment by janfjohannes** [comment]
@samuelstroschein  what is the exacct problem here, i am not sure i understand what fs inconsistency is triggered inside paraglide here?

**Comment by samuelstroschein** [comment]
@janfjohannes it is not an inconsistency. 

creating directories recursively is a pain with node fs and leads to constant bugs. {recursive: true} sometimes works  but only with a gizellion of edge cases like directory already exits = error stuff. 

i just want the following to work and don't think about "oh maybe inlang already exists or maybe inlang/messages already exists

```ts
await fs.createDirectory('/inlang/messages', { recursive: true })
```

**Comment by janfjohannes** [comment]
@samuelstroschein  ah got ya, as you bring up vscode as positive example again and that was also on my list, what do you think about evaluating using a subset of the vscode one. (@martin-lysk  you have any thought?)

**Comment by samuelstroschein** [comment]
@janfjohannes vscode fs api is not async and has no interop with upcoming filesystem types. hence, likely not a good idea.  

My gut feeling is that we can define our own API based on our requirements pretty well. Take the upcoming file types and stay close to the JS standard types like `File` and we should be good for the foreseeable future. But, I am keen to hold off for as long as possible. Unfortunately, we have daily fs bugs at this point due to implementation differences or because the node fs API is not "sane"

**Comment by janfjohannes** [comment]
> @janfjohannes vscode fs api is not async and has no interop with upcoming filesystem types. hence, likely not a good idea.

this one confuses me all vscode functions are thenable, so async no? https://github.com/Microsoft/vscode/blob/51a880315fd0ec2cafb511a17de48ec31802ba6d/src/vs/vscode.d.ts#L4968

also they have the concept of filesystem provider modules that plugin any filesystem type necessary, there allready is a memfs, nodejs with windows linux and mac normalization and a git backend as well as some experimental ones like opfs.

**Comment by samuelstroschein** [comment]
>this one confuses me all vscode functions are thenable, so async no? 

The API has been started in 2011. Well before async was a thing in JS. Hence, thenable is prob their async before async existed thing

**Comment by samuelstroschein** [comment]
Fixing this bug is incredible time consuming and somehow not possible https://www.loom.com/share/63338b8c025a422e8d78c26eadf2f6aa?sid=65eede94-9bb7-4955-9f22-fd5a816272fb

**Comment by samuelstroschein** [comment]
I hope the bug is fixed as part of #1624

**Comment by samuelstroschein** [comment]
closing because it should be fixed
